controllers/quickstart_constants.go

```diff
@@ -238,6 +238,54 @@ spec:
         failedTaskHelp: This task isn’t verified yet. Try the task again.
       summary:
         success: You have successfully created BucketClass
+        failed: Try the steps again. 
+    - title: Encrypt your StorageClass
+      description: |-
+
+          StorageClass encryption enables you to provide individual keys to each of your StorageClusters.
+
+
+          You can create an encrypted StorageClass while configuring your StorageCluster, or when creating a new StorageClass.
+
+
+          **To create an encrypted StorageClass while setting up your cluster:**
+
+          1. After installing OpenShift Data Foundation, select **Create StorageSystem**.
+
+          2. In the creation wizard, go to **Security and network** step.
+
+          3. Check the **StorageClass encryption** box.
+
+          4. Select your **Key management service provider**.
+
+          5. Provide your connection details.
+
+
+          **To create a new encrypted StorageClass:**
+
+          1. In the navigation menu, select **Storage > StorageClasses**, and select **Create StorageClass**.
+
+          2. In the **Provisioner** field, select **openshift-storage.rbd.csi.ceph.com**.
+
+          3. Check the **Enable Encryption** box.
+
+          4. Select your **Key management service provider**.
```

Comment:
```suggestion
          4. Select your **KMS provider**.
```

Reply:
same as above... `Key management service provider` is the exact name that we show in the UI.